src/py/flwr/cli/ls.py

```diff
@@ -325,11 +354,19 @@ def _display_one_run(
 
     run_dict = {run_id: run_from_proto(proto) for run_id, proto in res.run_dict.items()}
 
-    if output_format == CliOutputFormat.json:
+    if output_format == CliOutputFormat.JSON:
         Console().print_json(_to_json(_format_runs(run_dict, res.now)))
     else:
         Console().print(_to_table(_format_runs(run_dict, res.now)))
 
 
-def _print_json_error() -> None:
+def _print_json_error(msg: str, e: Union[Exit, SystemExit, Exception]) -> None:
     """Print error message as JSON."""
+    Console().print_json(
+        json.dumps(
+            {
+                "success": "false",
```

Comment:
```suggestion
                "success": False,
```

Reply:
Shouldn't it be a true `bool`?

Reply:
Good catch. When serialized, it'll be converted to JSON's `true` or `false`. Updated.